### ISSUE TITLE
Minecraft client crashing after connecting to limbo

### ISSUE THREAD

**Issue by dyanakiev** [Nan1t/NanoLimbo#1]
Minecraft client crashing after connecting to limbo

As you can see im trying to connect to lobby with is currently offline and im getting redirected to the limbo and my client crashes. Im using velocity, and minecraft client 1.16.4 with optifine.

```
[10:51:26 INFO]: [connected player] name (/127.0.0.1:53669) has connected
[10:51:26 ERROR]: [connected player] name (/127.0.0.1:53669): unable to connect to server lobby
io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: /127.0.0.1:25500
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Net.java:660) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:875) ~[?:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:707) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[velocity-1.1.2.jar:1.1.2]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[velocity-1.1.2.jar:1.1.2]
	at java.lang.Thread.run(Thread.java:832) [?:?]
[10:51:26 INFO]: [server connection] name -> limbo has connected
[10:51:29 INFO]: [connected player] name (/127.0.0.1:53669) has disconnected
[10:51:29 INFO]: [server connection] name -> limbo has disconnected
```

Minecraft crash report
```
java.lang.NullPointerException: Unexpected error
	at dzz.a(GameRenderer.java:987)
	at dzz.a(GameRenderer.java:693)
	at djz.e(SourceFile:1048)
	at djz.e(SourceFile:681)
	at net.minecraft.client.main.Main.main(SourceFile:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:159)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:30)
```

Limbo logs
```
[10:50:07] [INFO]: Starting server...
[10:50:07] [DEBUG]: Using Java NIO transport type
[10:50:07] [INFO]: Server started on localhost/127.0.0.1:65035
[10:50:30] [DEBUG]: Pinged from /127.0.0.1:53609
[10:50:30] [INFO]: Player name connected (/127.0.0.1:53609)
[10:50:30] [WARNING]: Undefined incoming packet: 0xb
[10:50:30] [WARNING]: Undefined incoming packet: 0xb
[10:50:30] [WARNING]: Undefined incoming packet: 0x0
[10:50:30] [WARNING]: Undefined incoming packet: 0x13
[10:50:30] [WARNING]: Undefined incoming packet: 0x25
[10:50:44] [WARNING]: Undefined incoming packet: 0x5
[10:50:44] [INFO]: Player name disconnected
[10:50:46] [DEBUG]: Pinged from /127.0.0.1:53626
[10:50:46] [INFO]: Player name connected (/127.0.0.1:53626)
[10:50:46] [WARNING]: Undefined incoming packet: 0xb
[10:50:48] [INFO]: Player name disconnected
[10:51:26] [DEBUG]: Pinged from /127.0.0.1:53671
[10:51:26] [INFO]: Player name connected (/127.0.0.1:53671)
[10:51:26] [WARNING]: Undefined incoming packet: 0xb
[10:51:29] [INFO]: Player name disconnected
```

**Comment by Nan1t** [comment]
This is Optifine support issue. Thanks for the report, it will be fixed as soon as possible.

**Comment by ToxiWoxi** [comment]
Any progress on that yet?

**Comment by Nan1t** [comment]
I found that this problem with dimension registry. But i have no time to fix it just now. Wait please, i'll fix it anyway.

**Comment by Nan1t** [comment]
Solution was easier than i thought. Fixed in version 1.2.2.
